src/pages/index.js

```diff
@@ -49,18 +102,25 @@ const cardSection = new Section(
   cardListEl
 );
 
-cardSection.renderItems();
-
 /** USER CLASS **/
 
-const user = new UserInfo(".profile__title", ".profile__description");
+const user = new UserInfo(
+  ".profile__title",
+  ".profile__description",
+  ".profile__avatar"
+);
 
 /** VALIDATOR CLASS **/
 
-const editFormValidator = new FormValidator(validationConfig, profileEditForm);
+const profileFormValidator = new FormValidator(
+  validationConfig,
+  profileEditForm
+);
+const avatarFormValidator = new FormValidator(validationConfig, avatorEditForm);
 const addFormValidator = new FormValidator(validationConfig, addCardForm);
```

Comment:
**COULD BE IMPROVED**

You can **universally** create instances of validators for all forms in the project storing them inside one object: `formValidators`.  Then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
// define an object for storing validators
const formValidators = {}

const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // Here you get the name of the form (if you don’t have it then you need to add it into each form in `index.html` first)
    const formName = formElement.getAttribute('name')

   // Here you store the validator using the `name` of the form
    formValidators[formName] = validator;
    validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```